.github/ISSUE_TEMPLATE/mod_crash.yml

```
@@ -1,21 +1,22 @@
-name: Mod Incompatibility
-description: Report an incompatible mod
-labels: [compat]
+name: Mod Crash
+description: Report mod(pack)s that crashes the game.
+labels: [compat, crash]
 body:
   - type: markdown
     attributes:
       value: "## Welcome!"
   - type: markdown
     attributes:
       value: |
-        Thank you for taking your time to report this incompatibility!
-        Make sure you are running the latest version of Connector and its dependencies before reporting.
+        Thank you for taking your time to report this incompatibility! You gotta make sure these before reporting...
```

Wording can be imporved here
```suggestion
        Thank you for taking your time to report this incompatibility! Before opening an issue, please make sure that:
```